pkg/gadgets/bindsnoop/tracer/core/tracer.go

```diff
@@ -177,7 +177,8 @@ func (t *Tracer) start() error {
 		return fmt.Errorf("error opening ipv6 kprobe: %w", err)
 	}
 
-	t.reader, err = perf.NewReader(t.objs.bindsnoopMaps.Events, os.Getpagesize())
+	t.reader, err = perf.NewReader(t.objs.bindsnoopMaps.Events,
+		gadgets.PerfBufferPages*os.Getpagesize())
```

Comment:
Why did you break just this line 😝 ?

Reply:
That's the first one I edited, then I chose not to break them.